tools/update-version/update-version.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python3
+# This script will to help to manage rhdm components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
```

Comment:
Can you please change update the script location with the correct path `tools/update-version/update-version.py` also calling it with `python3` and giving one more example with `--confirm` flag. Thanks.